### DISCUSSION TITLE
[FR] First-party Instructions field

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#8845] [Ideas]
### Description

Similar to this FR: https://github.com/craftcms/cms/issues/5230 but not restricted to Matrix fields. I'm currently using the Entry Instructions plugin too, but it would be great to have a field built into the control panel (or another solution) that's specifically for describing how an entry type works.

My use case is that I use Entry Types for different templates, so I normally add a field at the top of each entry type which explains what the entry type does, like automatically listing all the entries in a news section.

**Comment by brandonkelly** [comment]
A version of this is planned, but more as an enhancement to field layouts; not as its own field type.
